common-app-lib/src/cmr/common_app/services/search/parameter_validation.clj

```diff
@@ -229,7 +260,7 @@
 
 (def standard-valid-params
   "The set of standard valid query level parameters."
-  #{:page-size :page-num :offset :sort-key :result-format :options})
+  #{:offset :options :page-num :page-size :result-format :scroll :scroll-id :sort-key})
```

Comment:
I changed this to not add it to the params - this was forcing me to add scroll_id to the list of valid params, which we don't want. Instead I now add it to the context, and from there to the query model.